File: flexinput/src/main/java/com/lytefast/flexinput/managers/FileManager.java
package com.lytefast.flexinput.managers;

import android.content.Context;
import android.net.Uri;

import java.io.File;


/**
 * @author Sam Shih
 */
public interface FileManager {
  File newImageFile();

  Uri toFileProviderUri(final Context context, File file);
}

File: flexinput/src/main/java/com/lytefast/flexinput/managers/FileManager.kt
package com.lytefast.flexinput.managers

import android.content.Context
import android.net.Uri

import java.io.File


/**
 * @author Sam Shih
 */
interface FileManager {
  fun newImageFile(): File

  fun toFileProviderUri(context: Context, file: File): Uri
}
